mad-dog/src/helpers.js

```diff
@@ -119,12 +119,13 @@ async function _addUsers ({ userCount, executeAll, executeOne, existingUserIds,
           userId = newUserId
         }
 
+        // TODO: test to see if this is necessary
         // Wait 1 indexing cycle to get all proper and expected user metadata, as the starter metadata
         // does not contain all necessary fields (blocknumber, track_blocknumber, ...)
         await waitForIndexing()
         const userWalletAddress = getLibsWalletAddress(libs)
         const userAccount = await getUserAccount(libs, userWalletAddress)
-        setCurrentUserAndUpdateLibs(libs, userAccount) // might not need this anymore bc refactored signup flow
+        setCurrentUserAndUpdateLibs(libs, userAccount)
```

Comment:
seems to me like this is unnecessary, also pls remove comment on line 122

Reply:
what was your reasoning for not removing this earlier? i might be misreading. maybe easier to quick call

Reply:
resolved offline, removed